ResidentsSocialCarePlatformApi/V1/Controllers/ResidentsController.cs

```diff
@@ -90,7 +90,7 @@ public IActionResult ListCaseNotes(long personId)
         /// <response code="404">No visit information found for the specified Person ID</response>
         [ProducesResponseType(typeof(List<Boundary.Responses.VisitInformation>), StatusCodes.Status200OK)]
         [HttpGet]
-        [Route("{personId}/visits")]
+        [Route("{personId:long}/visits")]
         public IActionResult GetVisitInformation(long personId)
```

Comment:
For my own understanding, do we need to do this suppress a warning? Or it more a readability thing?

Reply:
It's more about being explicit. I think .net is able to figure out what we mean but without defining the value being taken from the route it expects the value to be a string by default. It then takes the string and converts it to a long, before our route would have accepted /residents/foo/visits though

By explicitly stating the route takes a long the controller won't accept any string given to it for the route parameter. In fact I think it would 404.

https://docs.microsoft.com/en-us/aspnet/web-api/overview/web-api-routing-and-actions/attribute-routing-in-web-api-2#constraints

Reply:
Got it, thanks for the link!